Lib/axisregistry/data/extrusion_depth.textproto

```diff
@@ -0,0 +1,9 @@
+# Extrusion Depth, based on https://github.com/justvanrossum/nabla
+tag: "EDPT"
+display_name: "Extrusion Depth"
+min_value: 0
+default_value: 0
+max_value: 1000
+precision: 0
+description:
+  "Controls extrustion depth."
```

Comment:
```suggestion
  "Controls extrusion depth."
```

Reply:
added in https://github.com/googlefonts/axisregistry/pull/61/commits/bd67f5b0a8c46a31ab14fcc608c63689334a6741